src/baseVisitor.ts

```diff
@@ -647,13 +646,21 @@ export class BaseVisitor<T> implements Required<RecursiveVisitors<T>> {
     }
   }
   TemplateElement = ignore
-  TemplateLiteral<S>(n: swc.TemplateLiteral, st: S, cb: Callback<S>) {
+  TemplateLiteral<S>(n: swc.TemplateLiteral | swc.TsTemplateLiteralType, st: S, cb: Callback<S>) {
     for (const quasis of n.quasis) {
       cb(quasis, st)
     }
 
-    for (const expressions of n.expressions) {
-      cb(expressions, st)
+    if ('expressions' in n) {
```

Comment:
Was `in` necessary for the type narrowing or would `if (n.expressions)` work too? If so I think I prefer the latter syntax.

Reply:
in is necessary in this case